store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -24,13 +34,25 @@
  */
 public class ExternalVolumeInfoHandler {
 
+    public static final String DIR_PATH = File.separator + "opt" + File.separator + "zimbra" + File.separator + "smconfig";
```

Comment:
Reduce the scope of constants if these are used in the same class only.

Reply:
Done